STYLE_GUIDE.md

```diff
@@ -272,6 +272,20 @@ end
   So set up it on the local rubocop config manually.
   <sup>[[link](#style-magic-link)]</sup>
 
+* <a name="style-braces-around-hash-params"></a>
+  We prefer to write hash parameters in style dependent to interfaces public API.
```

Comment:
eh, this still isn't clear.

Anyway in the [cop page](https://rubocop.readthedocs.io/en/latest/cops_style/#stylebracesaroundhashparameters), he have a good description for this rule

```
# The `context_dependent` style checks that the last parameter
# doesn't have braces around it, but requires braces if the
# second to last parameter is also a hash literal.
```

So let's re-phrase it and use it in our style guide 

```
Not to use braces around the last hash literal parameter. 
But requires braces if the second to last parameter is also a hash literal
```

Thoughts?

Reply:
Looks good to me